### ISSUE TITLE
Automate and improve paramater documenting process

### ISSUE THREAD

**Issue by dcwalk** [cosmos/gaia#1054]
## Summary

As part of the gov->gaia merge we now have a script that queries the parameters, and a way to display those parameters on hub.cosmos.network (https://hub.cosmos.network/main/governance/params-change/param_index.html).
This is an enhancement that builds on  #68 , PR to enable that post-merge is here: #1051. However there are some improvements and automation we can/should do to make the process less manual / prone to falling out of date 

## Problem Definition

However the process is manual / prone to falling out of date and so there are some improvements and automation we can/should do! 

## Proposal

We should:
- [ ] update the script to try a few nodes vs. one to be a little more future proof 
- [ ] update the script to query for the subspace keys instead of hardcoding them (post- https://github.com/cosmos/cosmos-sdk/pull/9884)
- [ ] integrate running the script + updating params into docs build process? (in makefile docs-build? in gh actions?)

____

#### For Admin Use

- [x] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by dcwalk** [comment]
Sam mentioned we using peers from chain registry: https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json

It looks like the new subspaces query currently unreleased in cosmos-sdk (#9884): https://github.com/cosmos/cosmos-sdk/blob/master/CHANGELOG.md   so we are blocked on that part until a release + in gaia

**Comment by nooomski** [comment]
Bot accidentally closed this. Opening again.

**Comment by okwme** [comment]
hey @dcwalk and @uditvira any updates on this issue? Should we icebox it or add #nice-to-have as a label or is it proressing?

**Comment by dcwalk** [comment]
I think we were waiting on an unreleased feature of cosmos-sdk: ``#9884 Provide a new gRPC query handler, /cosmos/params/v1beta1/subspaces, that allows the ability to query for all registered subspaces and their respective keys.`` 

Quickly checking I still see that as unreleased.  

This is a good candidate for #nice-to-have as the first version is committed to the docs :)

**Comment by okwme** [comment]
closed since https://github.com/cosmos/cosmos-sdk/issues/9884 was merged
